src/components/swap/SwapModalHeaderAmount.tsx

```diff
@@ -0,0 +1,74 @@
+import { formatCurrencyAmount, NumberType } from '@uniswap/conedison/format'
+import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
+import Column from 'components/Column'
+import CurrencyLogo from 'components/Logo/CurrencyLogo'
+import Row from 'components/Row'
+import Tooltip from 'components/Tooltip'
+import { ReactNode, useState } from 'react'
+import { Info } from 'react-feather'
+import styled, { useTheme } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const MAX_AMOUNT_STR_LENGTH = 9
+
+export const Label = styled.span`
+  color: ${({ theme }) => theme.textSecondary};
+  margin-right: 0.5rem;
```

Comment:
To be honest, I wasn't aware we're using `rem` up to this point. Apparently, we have around 150 occurrences of `rem` throughout the codebase vs ~2000 occurrences of `px`. Doesn't look like `rem` is the way forward, given our codebase is mostly filled with pixels. 

One of the things I would like to make sure, of while we're at this widget replacement work, is not to reduce code debt if we can do it while adding new features -> great.

I would suggest we discuss this at a meeting, which is the way we want to go forward and whether we want to refactor e.g. to pixels while copying the code over.